src/core/styles/components/time-picker/_time-picker-component.scss

```diff
@@ -0,0 +1,68 @@
+@include b(igx-time-picker) {
+
+    @include e(header) {
+        @extend %igx-time-picker__header !optional;
+    }
+
+    @include e(header-ampm) {
+        @extend %igx-time-picker__header-ampm !optional;
+    }
+
+    @include e(header-hour){
+        @extend %igx-time-picker__header-hour !optional;
+    }
+
+    // COLUMN
+    @include e(column) {
+        @extend %igx-time-picker__column !optional;
+    }
+
+    @include e(item) {
+        @extend %igx-time-picker__item !optional;
+    }
+
+    @include e(item, $mod: selected) {
+        @extend %igx-time-picker__item--selected !optional
+    }
+
+    @include e(item, $m: active) {
+        @extend %igx-time-picker__item--active !optional
+    }
+
+    // HOUR
+    @include e(hourList) {
+        @extend %igx-time-picker__hourList !optional
+    }
+
+
+    // MINUTE
+    @include e(minuteList) {
+        @extend %igx-time-picker__minuteList !optional
+    }
+
+    // AM PM
+    @include e(ampmList) {
+        @extend %igx-time-picker__ampmList !optional
+    }
+
+    @include e(body) {
+        @extend %igx-time-picker__body !optional;
+    }
+
+    .igx-dialog__window {
+        @extend %time-picker-display !optional;
+    }
+
+    .igx-dialog__window,
+    .igx-dialog__window-content {
+        @extend %time-picker-content !optional;
+    }
+
+    .igx-dialog__window-title {
+        @extend %time-picker-dialog-title !optional;
+    }
+
+    .igx-dialog__window-actions {
+        @extend %time-picker-dialog-actions !optional;
+    }
+}
```

Comment:
Update styles for vertical time-picker to be in sync with the new dialog styles. Remove unnecessary rules and placeholder classes.